components/molecules/Experiment.js

```diff
@@ -22,10 +22,13 @@ export const Experiment = (props) => {
     >
       <Link href={props.href}>
         <a
-          className="block text-p text-custom-blue-projects-link underline hover:opacity-70"
+          className="flex block text-p text-custom-blue-projects-link underline hover:opacity-70"
           tabIndex="0"
         >
           {props.title}
+          {props.url && props.url.substring(0, 8) === "https://" ? (
```

Comment:
Instead of adding another property, would it not be possible to just use the href prop that is already being passed in? Regardless of language the protocol should be the same.

Reply:
Yes, you are right. Thank you for point it out!